fenix/app/src/main/res/values/strings.xml

```diff
@@ -65,6 +65,10 @@
        Common myths about private browsing
     </string>
 
+    <!-- Total Private Browsing Mode -->
+    <string name="felt_privacy_desc_card_title" translatable="false">Leave no traces on this device.</string>
```

Comment:
We need to have a localization note comment for each strings to provide context around the string for our translators.